### DISCUSSION TITLE
Action modal create data to another resource

### DISCUSSION THREAD

**Discussion by stefantheo** [filamentphp/filament#9378] [Help]
Hi, i'm new to Filament PHP,
I have 2 resources/models ('InvoiceResource', 'ReceiptResource')
In my InvoiceResource i have:

>   ->actions([
>                 ActionGroup::make([
>                     Action::make('RegisterPay')
>                         ->hidden(fn(Invoice $record) => $record->status != InvoiceStatus::Activa)
>                         ->modalHeading(fn(Invoice $record) => Register  ' . $record->full_number)
>                         ->form([
>                             TextInput::make('receipts.amount_paid')
>                                 ->required()
>                                 ->numeric(16, 4),
>                             Select::make('receipts.receipt_serie_id')
>                                 ->searchable()
>                                 ->preload()
>                                 ->default(function () {
>                                     $serie = ReceiptSerie::query()
>                                         ->where('is_default', true)
>                                         ->first('id');
>                                     return $serie->id;
>                                 })
>                                 ->relationship('receipt_serie', 'serie')
>                                 ->disablePlaceholderSelection()
>                                 ->required(),
>                             Forms\Components\TextInput::make('receipts.number')
>                                 ->required()
>                                 ->required(),
>                             DatePicker::make('receipts.date')
>                                 ->displayFormat('d.m.Y')
>                                 ->default(now())
>                             TextInput::make('receipts.transaction_id')
>                             Forms\Components\Textarea::make('receipts.note')
>                             TextInput::make('receipts.invoice_id')
>                                 ->hidden()
>                                 ->default(function ($record) {
>                                     return $record->id;
>                                 })
>                         ])
>                         ->action(function (Receipt $receipt, array $data): void{
>                           $receipt->amount_paid = $data['receipts']['amount_paid'];
>                           $receipt->receipt_serie_id = $data['receipts']['receipt_serie_id'];
>                           $receipt->number = $data['receipts']['number'];
>                           $receipt->date = $data['receipts']['date'];
>                           $receipt->transaction_id = $data['receipts']['transaction_id'];
>                           $receipt->note = $data['receipts']['note'];
>                           $receipt->invoice_id = $data['receipts']['invoice_id'];
>                           $receipt->save();
>                         })
>                         ->color('success')
>                         ->icon('heroicon-m-check-badge')
>                         ->slideOver(),

I want to add data from InvoiceResource modal action form to my ReceiptResource, when i complete the form, i got an error that says cannot find "receipt_serie_id" in my InvoiceResource, how i can fix that?

**Comment by leandrocfe** [comment]
I think if you want to save manual data, you should add `->dehydrated(true)` in the `Select`

**Reply by stefantheo** [reply]
Same error "SQLSTATE[42S22]: Column not found: 1054 Unknown column 'receipt_serie_id' in 'field list'"

**Reply by leandrocfe** [reply]
Sorry I'm on mobile. 

Why not a [relation manager](https://filamentphp.com/docs/3.x/panels/resources/relation-managers/) ?
